superbench/benchmarks/micro_benchmarks/cublas_function/cublas_helper.cpp

```diff
@@ -0,0 +1,215 @@
+ * @param  c                output matrix
+ */
+void sgemm(cublasHandle_t handle, int transa, int transb, int m, int n, int k, const float *a, const float *b,
+           float *c) {
+    float alpha = 1.0f;
+    float beta = 1.0f;
+    CUBLAS_SAFE_CALL(cublasSgemm(handle, (transa ? CUBLAS_OP_T : CUBLAS_OP_N), (transb ? CUBLAS_OP_T : CUBLAS_OP_N), m,
+                                 n, k, &alpha, a, (transa ? k : m), b, (transb ? n : k), &beta, c, m));
+}
+
+/**
+ * @brief                   cublas function of gemm, wrapper of cublasCgemm
+ * @param  handle           cublas handle
+ * @param  transa           whether matrixA transpose
+ * @param  transb           whether matrixB transpose
+ * @param  m                m of matrix m*n,n*k
+ * @param  n                n of matrix m*n,n*k
+ * @param  k                k of matrix m*n,n*k
+ * @param  a                input matrixA
+ * @param  b                input matrixB
+ * @param  c                output matrix
+ */
+void cgemm(cublasHandle_t handle, int transa, int transb, int m, int n, int k, const cuComplex *a, const cuComplex *b,
+           cuComplex *c) {
+    cuComplex alpha = make_cuComplex(1.0f, 0.0f);
+    cuComplex beta = make_cuComplex(0.0f, 0.0f);
+    CUBLAS_SAFE_CALL(cublasCgemm(handle, (transa ? CUBLAS_OP_T : CUBLAS_OP_N), (transb ? CUBLAS_OP_T : CUBLAS_OP_N), m,
+                                 n, k, &alpha, a, (transa ? k : m), b, (transb ? n : k), &beta, c, m));
+}
+
+/**
+ * @brief                   cublas function of GemmEx, wrapper of cublasGemmEx
+ * @param  handle           cublas handle
+ * @param  transa           whether matrixA transpose
+ * @param  transb           whether matrixB transpose
+ * @param  m                m of matrix m*n,n*k
+ * @param  n                n of matrix m*n,n*k
+ * @param  k                k of matrix m*n,n*k
+ * @param  a                input matrixA
+ * @param  b                input matrixB
+ * @param  c                output matrix
+ * @param  type             matrix type, 'float' or 'half'
+ * @param  use_tensor_core  whether use tensor core
+ */
+void gemmEx(cublasHandle_t handle, int transa, int transb, int m, int n, int k, const void *a, const void *b, void *c,
+            std::string type, bool use_tensor_core) {
+    float alpha = 1.0f;
+    float beta = 0.0f;
+    cublasComputeType_t compute_type = CUBLAS_COMPUTE_32F;
+    cudaDataType_t matrix_type;
+    cublasGemmAlgo_t algo;
+    algo = (use_tensor_core ? CUBLAS_GEMM_DFALT_TENSOR_OP : CUBLAS_GEMM_DFALT);
+    if (type.compare("float")) {
+        matrix_type = CUDA_R_32F;
+        CUBLAS_SAFE_CALL(cublasGemmEx(handle, (transa ? CUBLAS_OP_T : CUBLAS_OP_N),
+                                      (transb ? CUBLAS_OP_T : CUBLAS_OP_N), m, n, k, &alpha, a, matrix_type,
+                                      (transa ? k : m), b, matrix_type, (transb ? n : k), &beta, c, matrix_type, m,
+                                      compute_type, algo));
+    }
+    if (type.compare("half")) {
```

Comment:
else if

Reply:
thank you for the comments, I have revised it and the following